app/test_main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+from unittest import mock
+import pytest
+from app.main import can_access_google_page
```

Comment:
there should be an empty blank lane between built-in modules and local

Reply:
agree, according to pep8 
https://peps.python.org/pep-0008/#module-level-dunder-names:~:text=Imports%20should%20be,group%20of%20imports.